app/src/main/java/com/example/yeseul/movieapp/view/main/MainActivity.java

```diff
@@ -108,14 +108,15 @@ public void onSearchResultEmpty(String searchKey) {
     }
 
     /**
-     * 영화 상세 정보 URL 로 연결 */
+     * 영화 줄거리 표시
+     */
     @Override
-    public void startMovieDetailPage(String linkUrl) {
-
-        CustomTabsIntent customTabsIntent = new CustomTabsIntent.Builder()
-                .setToolbarColor(getResources().getColor(R.color.colorPrimary))
-                .build();
-
-        customTabsIntent.launchUrl(this, Uri.parse(linkUrl));
+    public void startMovieDetailPage(String movieStory, int position) {
+        if(movieStory != null) {
```

Comment:
String 클래스의 isEmpty 메서드를 사용하면 null과 빈 문자열을 한번에 검사할수있습니다

Reply:
```suggestion
 if(movieStory.isEmpty()) {
```
를 말씀해주신건가요
String.isEmpty는 String이 null일시에 NPE 가 발생해서 자바상에서 빈 문자열을 한번에 검사한다면
```suggestion
 if(TextUtils.isEmpty(movieStory)) {
```
처럼 TextUtils 를 써야할것 같습니다

Reply:
오우 감사합니다!!